### PR TITLE
Edit: getting started

### PR DESCRIPTION
**Pull request by Mocab**
This guide might be outdated, I am planning to separate this guide into multiple guides/categories in the near future. It will be updated by then.

### CODE REVIEW

File: _posts/minecraft/general/2021-08-31-getting-started.md [hunk 1]
```diff
@@ -74,6 +73,14 @@ These guides give a detailed explanation of all the settings they do on your ser
 
 You also have a shortcut. You can check out premade configs by clicking [here](https://github.com/flaxeneel2/pterodactyl-optimized-paper-egg) and then edit the values to your liking (PaperMC).
 
+**Startup Flags**
+The startup flags are also very important for optimization and makes a major difference in performance.
+1. Go to your server panel
+2. Go to startup tab
+3. Scroll down to JAVA ARGUMENTS
+4. Paste the flags [here](https://haste.flaxeneel2.net/Z3SgPnxvQyaJ9Q47LPbLooXKHq7FSax6UBFkBzpY03OOCdV22k.coffeescript) in it
+5. Start your server and you are done!
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-08-31-getting-started.md:82]: Just telling the user to paste in something from a website and starting their server isn't something I want to do.
If you're going to add in something about startup flags, could you at least explain what each flag does, or explain what all the startup flags combined will do to the server.

I'm also fine with separating this to a new article.

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

File: _posts/minecraft/general/2021-08-31-getting-started.md (hunk 1 above)
Comment (Mocab) [reply _posts/minecraft/general/2021-08-31-getting-started.md:82]: > I'm also fine with separating this to a new article.

Do you think its a good idea to sort it like this?
/minecraft/startup/java-arguments

Comment (DaddyChimken) [reply _posts/minecraft/general/2021-08-31-getting-started.md:82]: Not sure myself. @itsjuoum ^^

Comment (Mocab) [reply _posts/minecraft/general/2021-08-31-getting-started.md:82]: ive been brainstorming for a while, but i cant come up with a better one

**Comment by KorbsStudio** [comment]
At least change the file name from **2021-08-31-getting-started.md** to **2021-09-04-getting-started.md**

**Comment by Mocab** [comment]
> At least change the file name from **2021-08-31-getting-started.md** to **2021-09-04-getting-started.md**

As I said above, I have not updated the guide other than the java flags, I will rename it once it has been updated.

File: _posts/minecraft/general/2021-08-31-getting-started.md (hunk 1 above)
Comment (itsjuoum) [reply _posts/minecraft/general/2021-08-31-getting-started.md:82]: Yeah, I think that it's better if we separate this to a new article

Comment (Mocab) [reply _posts/minecraft/general/2021-08-31-getting-started.md:82]: What should the permalink be?

Comment (itsjuoum) [reply _posts/minecraft/general/2021-08-31-getting-started.md:82]: This isn't a discussion for that article, focus on this one first.

File: _posts/minecraft/general/2021-08-31-getting-started.md [hunk 2]
```diff
@@ -74,13 +74,17 @@ These guides give a detailed explanation of all the settings they do on your ser
...
 2. Go to startup tab
-3. Scroll down to JAVA ARGUMENTS
+3. Scroll down to `JAVA ARGUMENTS`
 4. Paste the flags [here](https://haste.flaxeneel2.net/Z3SgPnxvQyaJ9Q47LPbLooXKHq7FSax6UBFkBzpY03OOCdV22k.coffeescript) in it
 5. Start your server and you are done!
 
+> What does the flags do? 
+it pre processes the memory to be more usable
+
+
```
Comment (itsjuoum) [inline _posts/minecraft/general/2021-08-31-getting-started.md:87]: There are some typos here, but I'm also unsure if this info is correct. @DaddyChimken can you please confirm?

Comment (DaddyChimken) [reply _posts/minecraft/general/2021-08-31-getting-started.md:87]: "it pre processes the memory to be more usable"

No one will understand this, please fix the grammar and add more information while making it easier to understand.

File: _posts/minecraft/general/2021-08-31-getting-started.md [hunk 3]
```diff
@@ -74,6 +73,18 @@ These guides give a detailed explanation of all the settings they do on your ser
...
+**Startup Flags**
+Startup flags are also very important for optimization and may make a major difference in performance.
+1. Go to your server panel
+2. Go to startup tab
+3. Scroll down to `JAVA ARGUMENTS`
+4. Paste the flags [here](https://haste.flaxeneel2.net/Z3SgPnxvQyaJ9Q47LPbLooXKHq7FSax6UBFkBzpY03OOCdV22k.coffeescript) in it
+5. Start your server and you are done!
+
+> What does the flags do? 
+The flags optimize the ram to be more efficient.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-08-31-getting-started.md:85]: What flags are you talking about? How does it "optimize the ram"? How does using these flags make my server more "efficient"?

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

File: _posts/minecraft/general/2021-08-31-getting-started.md [hunk 4]
```diff
@@ -74,6 +73,18 @@ These guides give a detailed explanation of all the settings they do on your ser
...
 
+**Startup Flags**
+Startup flags are also very important for optimization and may make a major difference in performance.
+1. Go to your server panel
+2. Go to startup tab
+3. Scroll down to `JAVA ARGUMENTS`
+4. Paste the flags [here](https://haste.flaxeneel2.net/Z3SgPnxvQyaJ9Q47LPbLooXKHq7FSax6UBFkBzpY03OOCdV22k.coffeescript) in it
+5. Start your server and you are done!
+
+> What does the flags do?
```
Comment (itsjuoum) [inline _posts/minecraft/general/2021-08-31-getting-started.md:84]:
```suggestion
> What do the flags do? 
```

File: _posts/minecraft/general/2021-08-31-getting-started.md [hunk 5]
```diff
@@ -74,6 +73,18 @@ These guides give a detailed explanation of all the settings they do on your ser
...
+**Startup Flags**
+Startup flags are also very important for optimization and may make a major difference in performance.
+1. Go to your server panel
+2. Go to startup tab
+3. Scroll down to `JAVA ARGUMENTS`
+4. Paste the flags [here](https://haste.flaxeneel2.net/Z3SgPnxvQyaJ9Q47LPbLooXKHq7FSax6UBFkBzpY03OOCdV22k.coffeescript) in it
+5. Start your server and you are done!
+
+> What does the flags do? 
+You can find a detailed guide on [aikar's website](https://aikar.co/2018/07/02/tuning-the-jvm-g1gc-garbage-collector-flags-for-minecraft/).
```
Comment (itsjuoum) [inline _posts/minecraft/general/2021-08-31-getting-started.md:85]: I don't think that naming another person's website is a good thing to do. That website also takes a [bunch of time to load](https://i.imgur.com/avDbOvb.png).

**Review by itsjuoum — CHANGES_REQUESTED** [review]

File: _posts/minecraft/general/2021-08-31-getting-started.md (hunk 5 above)
Comment (Mocab) [reply _posts/minecraft/general/2021-08-31-getting-started.md:85]: I'll rename it to something else, as for the second issue, nothing I can do about, I am not a java Dev so I don't know what flags do in detail

File: _posts/minecraft/general/2021-08-31-getting-started.md [hunk 6]
```diff
@@ -74,6 +73,18 @@ These guides give a detailed explanation of all the settings they do on your ser
...
+**Startup Flags**
+Startup flags are also very important for optimization and may make a major difference in performance.
+1. Go to your server panel
+2. Go to startup tab
+3. Scroll down to `JAVA ARGUMENTS`
+4. Paste the flags [here](https://haste.flaxeneel2.net/Z3SgPnxvQyaJ9Q47LPbLooXKHq7FSax6UBFkBzpY03OOCdV22k.coffeescript) in it
+5. Start your server and you are done!
+
+> What do the flags do? 
+You can find a detailed guide [here](https://aikar.co/2018/07/02/tuning-the-jvm-g1gc-garbage-collector-flags-for-minecraft/).
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-08-31-getting-started.md:85]:
```suggestion
These flags help keep your server run with a stable TPS and without any large garbage collection spikes by tweaking different allocation and garbage collection settings for Minecraft. 
For further information on the flags, check out [this blog](https://aikar.co/2018/07/02/tuning-the-jvm-g1gc-garbage-collector-flags-for-minecraft).
```
Made this pretty quickly, feel free to correct any grammar mistakes.

Comment (DaddyChimken) [reply _posts/minecraft/general/2021-08-31-getting-started.md]: Now that I've re-read my suggestion, it sounds a bit off. I'll see if I can make any improvements tomorrow.

**Comment by KorbsStudio** [comment]
Heads up, this article was recently updated on my end for the Bedrock Edition.

File: _posts/minecraft/general/2021-08-31-getting-started.md [hunk 7]
```diff
@@ -74,6 +73,19 @@ These guides give a detailed explanation of all the settings they do on your ser
...
+Startup flags are also very important for optimization and may make a major difference in performance.
+1. Go to your server panel
+2. Go to startup tab
+3. Scroll down to `JAVA ARGUMENTS`
+4. Paste the flags [here](https://haste.flaxeneel2.net/Z3SgPnxvQyaJ9Q47LPbLooXKHq7FSax6UBFkBzpY03OOCdV22k.coffeescript) in it
+5. Start your server and you are done!
+
+> What do the flags do? 
+These flags help keep your server run with a stable TPS and without any large garbage collection spikes by tweaking different allocation and garbage collection settings for Minecraft. 
+For further information on the flags, check out [this blog](https://aikar.co/2018/07/02/tuning-the-jvm-g1gc-garbage-collector-flags-for-minecraft).
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-08-31-getting-started.md:86]:
```suggestion
> Why should I use these startup flags? 
These flags help keep your server run with a stable TPS and without any large garbage collection spikes by tweaking different allocation and garbage collection settings for Minecraft. 
For further information on these flags, check out [this blog](https://aikar.co/2018/07/02/tuning-the-jvm-g1gc-garbage-collector-flags-for-minecraft).
```

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

File: _posts/minecraft/general/2021-08-31-getting-started.md [hunk 8]
```diff
@@ -74,6 +73,19 @@ These guides give a detailed explanation of all the settings they do on your ser
...
+**Startup Flags**
+Startup flags are also very important for optimization and may make a major difference in performance.
+1. Go to your server panel
+2. Go to startup tab
+3. Scroll down to `JAVA ARGUMENTS`
+4. Paste the flags [here](https://haste.flaxeneel2.net/Z3SgPnxvQyaJ9Q47LPbLooXKHq7FSax6UBFkBzpY03OOCdV22k.coffeescript) in it
+5. Start your server and you are done!
+
+> Why should I use these startup flags? 
+These flags help keep your server run with a stable TPS and without any large garbage collection spikes by tweaking different allocation and garbage collection settings for Minecraft.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-08-31-getting-started.md:85]:
```suggestion
These flags help keep your server running with a stable TPS and without any large garbage collection spikes by tweaking different allocation and garbage collection settings for Minecraft. 
```

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

File: _posts/minecraft/general/2021-08-31-getting-started.md (hunk 8 above)
Comment (DaddyChimken) [reply _posts/minecraft/general/2021-08-31-getting-started.md:85]: Actually, nvm.

Comment (DaddyChimken) [reply _posts/minecraft/general/2021-08-31-getting-started.md:85]:
```suggestion
These flags help your server run with a stable TPS and without any large garbage collection spikes by tweaking different allocation and garbage collection settings for Minecraft. 
```

Comment (DaddyChimken) [reply _posts/minecraft/general/2021-08-31-getting-started.md:85]: This still seems a bit wordy and big imo, anyone have any suggestions?

Comment (Mocab) [reply _posts/minecraft/general/2021-08-31-getting-started.md:85]: cant be helped ¯\_(ツ)_/¯

Comment (DaddyChimken) [reply _posts/minecraft/general/2021-08-31-getting-started.md:85]: The sentence is super long with 2 "ands", which also makes it wordy and confusing to understand. 
*I also made this sentence in ~5 minutes, it definitely should be at least edited.*

Comment (Mocab) [reply _posts/minecraft/general/2021-08-31-getting-started.md:85]: We should remove ` and without any large garbage collection spikes` , im pretty sure most users don't know what it is, plus its the part of sentence that's confusing.

Comment (DaddyChimken) [reply _posts/minecraft/general/2021-08-31-getting-started.md:85]: Sure, I'm fine with that.

```suggestion
These flags help keep your server run with a stable TPS by tweaking different allocation and garbage collection settings for Minecraft. 
```

**Comment by DaddyChimken** [comment]
Please rebase, there are merge conflicts.

**Comment by Mocab** [comment]
I hope I did it correctly 😅

**Comment by KorbsStudio** [comment]
There are no longer conflicts with this PR, will merge once approved.

**Comment by Mocab** [comment]
Ill close this pull request and start again, since this place is a mess
